.github/workflows/flutter-build.yml

```diff
@@ -3,11 +3,13 @@ on:
   push:
     paths:
       - '.github/workflows/**'
+      - 'api/**'
```

Comment:
will the Flutter build depend on API changes?

Reply:
The API directory defines both the service and also the data structures.  The Flutter client hand-rolls the JSON but it should use for example protoc-generated dart classes: https://developers.google.com/protocol-buffers/docs/darttutorial